tests/saliency_tests/visualizer_tests/test_image_visualizer.py

```diff
@@ -0,0 +1,55 @@
+import os
+import sys
+
+import numpy
+import pytest
+
+
+from chainer_chemistry.saliency.visualizer.image_visualizer import ImageVisualizer  # NOQA
+
+
+is_python_version2 = sys.version_info[0] < 3
+
+
+@pytest.mark.skipif(is_python_version2,
+                    reason='matplotlib configuration is necessary with'
+                           'python version 2')
+def test_image_visualizer(tmpdir):
+    # Only test file is saved without error
+    ch = 3
+    h = 32
+    w = 32
+    saliency = numpy.random.uniform(0, 1, (ch, h, w))
+    visualizer = ImageVisualizer()
+
+    # 1. test with setting save_filepath
+    save_filepath = os.path.join(str(tmpdir), 'tmp.png')
+    visualizer.visualize(saliency, save_filepath=save_filepath)
+    assert os.path.exists(save_filepath)
+
+    # 2. test with `save_filepath=None` runs without error
+    image = numpy.random.uniform(0, 1, (ch, h, w))
+    visualizer.visualize(
```

Comment:
Visualilze method stops unit testing.
Can you insert `plt.ion()` and `plt.close()`? like this.
```
    plt.ion()
    visualizer.visualize(
        saliency, save_filepath=None, feature_names=['hoge', 'huga', 'piyo'],
        num_visualize=2)
    plt.close()
```